src/transactions/transferFeedUtils.ts

```diff
@@ -400,3 +427,52 @@ export function isTransferTransaction(
 ): tx is TransferItemFragment {
   return (tx as TransferItemFragment).address !== undefined
 }
+
+// Note: This hook is tested from src/transactions/feed/TransferFeedItem.test.ts
+function useFiatConnectTransferDisplayInfo({ amount, transactionHash }: TokenTransfer) {
+  const { t } = useTranslation()
+  const tokenInfo = useTokenInfo(amount.tokenAddress)
+  const fcTransferDetails = useSelector(getCachedFiatConnectTransferSelector(transactionHash))
+  const cachedFiatAccountUses = useSelector(cachedFiatAccountUsesSelector)
+  const localCurrency = useSelector(getLocalCurrencyCode)
+  const account = useMemo(
+    () =>
+      cachedFiatAccountUses.find(
+        (account) => account.fiatAccountId === fcTransferDetails?.fiatAccountId
+      ),
+    [cachedFiatAccountUses, fcTransferDetails]
+  )
+  if (!account || !fcTransferDetails) {
+    return
+  }
+
+  const receive = Number(fcTransferDetails.quote.fiatAmount)
+  const total = Number(amount.value)
+  const exchangeRate = String(receive / total)
+  const sign = fcTransferDetails.quote.transferType === TransferType.TransferOut ? -1 : 1
+  const localAmount: LocalAmount = {
+    value: new BigNumber(fcTransferDetails.quote.fiatAmount).multipliedBy(sign),
```

Comment:
```suggestion
    value: new BigNumber(receive).multipliedBy(sign),
```

Reply:
also why are you using a bignumber here but not for exchangeRate?

Reply:
That is how the `LocalAmount` type is [defined](https://github.com/valora-inc/wallet/blob/266b2b56658e3d2f5d740d9f6588be19d564f87e/src/transactions/types.ts#L113), which is used [here](https://github.com/valora-inc/wallet/blob/266b2b56658e3d2f5d740d9f6588be19d564f87e/src/transactions/feed/TransferFeedItem.tsx#L61) to render the local currency in each `TransferFeedItem`. I'm not sure why `LocalAmount` was defined with exchangeRate as a string